languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/grammar.xml

```diff
@@ -10069,6 +10069,67 @@ though, and has since been slightly modified)-->
     </rulegroup>
   </category>
   <category id="CAT_GRAMMAIRE" name="Grammaire">
+      <rulegroup id="QUELLE_QUE" name="quelle que" default="temp_off">
+          <rule>
+              <pattern>
+                  <marker>
+                      <token regexp="yes">qu(&apostrophe;)?</token>
+                      <token>elle</token>
+                  </marker>
+                  <token regexp="yes">que|qu(&apostrophe;)</token>
+              </pattern>
```

Comment:
Now that you've added the fifth subrule for "qu'elle qu'un → quelqu'un", you need to make sure this rule ([1]) doesn't match where [5] is supposed to match (it needs an AP for "qu'elle qu'un").